### PR TITLE
QA8-1289 add corner tests for image (TC-132,TC-134,TC-136,TC-138)

### PR DESCRIPTION
**Pull request by Mykhaylyak**

### CODE REVIEW

File: test/extended/image.js [hunk 1]
```diff
@@ -29,9 +29,40 @@ describe('Image field suit', function () {
...
             let imageDisplayed = $(sel.imageDisplayed);
             expect(imageDisplayed).toBeDisplayed();
         });
     });
+    describe('Corner cases', function () {
+
+        it('TC-132 Verify that max size JPEG image 3.9 Mb by clicking', function () {
+            uploadImage(images.JPG_3);
+            let imageDisplayed = $(sel.imageDisplayed);
+            expect(imageDisplayed).toBeDisplayed()
```
Comment (Alex-Mashchenko) [inline test/extended/image.js:42]: in the end your string  use ";" [42]

File: test/extended/image.js [hunk 2]
```diff
@@ -29,9 +29,40 @@ describe('Image field suit', function () {
...
+
+        });
+
+        it('TC-134 Verify that max size PNG image 3.9 Mb by clicking', function () {
+            uploadImage(images.PNG_3);
+            let imageDisplayed = $(sel.imageDisplayed);
+            expect(imageDisplayed).toBeDisplayed()
```
Comment (Alex-Mashchenko) [inline test/extended/image.js:49]: in the end your string  use ";" [49]

File: test/extended/image.js [hunk 3]
```diff
@@ -29,9 +29,40 @@ describe('Image field suit', function () {
...
+
+        });
+
+        it('TC-136 Verify that max size JPEG image 1 Mb by clicking', function () {
+            uploadImage(images.JPG_1);
+            let imageDisplayed = $(sel.imageDisplayed);
+            expect(imageDisplayed).toBeDisplayed()
```
Comment (Alex-Mashchenko) [inline test/extended/image.js:56]: in the end your string  use ";" [56]

File: test/extended/image.js [hunk 4]
```diff
@@ -29,9 +29,40 @@ describe('Image field suit', function () {
...
+
+        });
+
+        it('TC-138 Verify that max size PNG image 1 Mb by clicking', function () {
+            uploadImage(images.PNG_1);
+            let imageDisplayed = $(sel.imageDisplayed);
+            expect(imageDisplayed).toBeDisplayed()
```
Comment (Alex-Mashchenko) [inline test/extended/image.js:63]: in the end your string  use ";" [63]

File: test/extended/image.js [hunk 5]
```diff
@@ -29,9 +29,40 @@ describe('Image field suit', function () {
...
+
```
Comment (Alex-Mashchenko) [inline test/extended/image.js:64]: delete empty line [64]

File: test/extended/image.js [hunk 6]
```diff
@@ -29,9 +29,40 @@ describe('Image field suit', function () {
...
+        });
+
```
Comment (Alex-Mashchenko) [inline test/extended/image.js:66]: delete empty line [66]

**Review by Alex-Mashchenko — CHANGES_REQUESTED** [review]

File: test/extended/image.js (hunk 1 above)
Comment (Mykhaylyak) [reply test/extended/image.js]: done

File: test/extended/image.js (hunk 2 above)
Comment (Mykhaylyak) [reply test/extended/image.js]: done

File: test/extended/image.js (hunk 3 above)
Comment (Mykhaylyak) [reply test/extended/image.js]: done

File: test/extended/image.js (hunk 5 above)
Comment (Mykhaylyak) [reply test/extended/image.js]: done

**Comment by Mykhaylyak** [comment]
delete unnecessary lines, add “;”  and renamed files

**Review by MariiaSosedova — CHANGES_REQUESTED** [review]

**Pull request closed without merging**